src/ert/enkf_main.py

```diff
@@ -43,7 +43,9 @@ def _backup_if_existing(path: Path) -> None:
 
 
 def _value_export_txt(
-    run_path: Path, export_base_name: str, values: Mapping[str, Mapping[str, float]]
+    run_path: Path,
+    export_base_name: str,
+    values: Mapping[str, Mapping[str, float | str]],
```

Comment:
All these function that are now accepting strings here, is that covered in tests?

Reply:
We don't have any unit tests for `_value_export_*` functions. Maybe we should....